src/commands/get_balance.ts

```diff
@@ -2,14 +2,19 @@ import { cliWalletDao } from '..';
 import { CLICommand, ParametersHelper } from '../CLICommand';
 import { AddressParameter, SeedPhraseParameter, WalletFileParameter } from '../parameter_declarations';
 
+const anArInWinston = 1_000_000_000_000;
+const numberOfDigitsAfterComma = 8;
```

Comment:
Why limit this to 8 digits? There's 10^12 Winston per AR. Therefore we need 12 decimal places for precision.